### PR TITLE
fix: change module name from cache to github.com/hertz-contrib/cache

### LINKED ISSUE #2: fix: wrong go module name
**Describe the bug**

**To Reproduce**

use command `go get github.com/hertz-contrib/cache`

**Expected behavior**

install successully

**Screenshots**

image

**Kitex version:**

Please provide the version of Kitex you are using.

**Environment:**
```
set GO111MODULE=on
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\27371\AppData\Local\go-build
set GOENV=C:\Users\27371\AppData\Roaming\go\env
set GOEXE=.exe
set GOEXPERIMENT=
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=D:\workspace\Projects\GolandProjects\pkg\mod
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=D:\workspace\Projects\GolandProjects
set GOPRIVATE=
set GOPROXY=https://goproxy.cn,direct
set GOROOT=D:\Go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=D:\Go\pkg\tool\windows_amd64
set GOVCS=
set GOVERSION=go1.19
set GCCGO=gccgo
set GOAMD64=v1
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=D:\workspace\Projects\GolandProjects\src\mini_tiktok\go.mod
set GOWORK=
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=C:\Users\27371\AppData\Local\Temp\go-build1065160713=/tmp/go-build -gno-record-gcc-switches
```

**Additional context**

### PR DESCRIPTION
**Pull request by wsrer**
#### What type of PR is this?
fix

#### What this PR does / why we need it (en: English/zh: Chinese):

en: 
i can't directly use command `go get github.com/hertz-contrib/cache` to use this repo
zh:

#### Which issue(s) this PR fixes:
[Fixes #2](https://github.com/hertz-contrib/cache/issues/2)

### CODE REVIEW

**Comment by li-jin-gou** [comment]
thanks and need to sign cla

File: cache.go [hunk 1]
```diff
@@ -41,10 +41,10 @@
 package cache
 
 import (
-	"cache/persist"
 	"context"
 	"encoding/gob"
 	"errors"
+	"github.com/hertz-contrib/cache/persist"
```
Comment (L2ncE) [inline cache.go]: sort import plz

eg.
```go
        "context"
	"errors"
// ...

	"github.com/hertz-contrib/cache/persist"
// ...
```

**Comment by L2ncE** [comment]
delete useless blank @li-jin-gou

File: cache.go (hunk 1 above)
Comment (li-jin-gou) [reply cache.go]: done

**Review by li-jin-gou — APPROVED** [review]

**Review by L2ncE — APPROVED** [review]
LGTM

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**